ui/lib/core/addon/templates/components/replication-actions.hbs

```diff
@@ -2,14 +2,29 @@
   <LayoutLoading />
 {{else}}
   <MessageError @errors={{errors}} />
-  {{#each (if selectedAction (array selectedAction) (replication-action-for-mode replicationMode model.replicationAttrs.modeForUrl)) as |replicationAction index|}}
-    <div class="box is-fullwidth is-marginless {{if (gt index 0) 'is-bottomless' 'is-shadowless'}}">
-      {{component (concat 'replication-action-' replicationAction)
-        onSubmit=(action "onSubmit")
-        replicationMode=replicationMode
-        model=model
-        replicationDisplayMode=replicationDisplayMode
-      }}
-    </div>
-  {{/each}}
-{{/if}}
+  {{#if selectedAction}}
+    {{#each (array selectedAction) as |replicationAction index|}}
+      <div class="box is-fullwidth is-marginless {{if (gt index 0) 'is-bottomless' 'is-shadowless'}}">
+        {{component (concat 'replication-action-' replicationAction)
+          onSubmit=(action "onSubmit")
+          replicationMode=replicationMode
+          model=model
+          replicationDisplayMode=replicationDisplayMode
+        }}
+      </div>
+    {{/each}}
+  {{else}}
+  <div class="replication-actions-grid-layout">
+    {{#each (if clusterMode (replication-action-for-mode replicationMode clusterMode) (replication-action-for-mode replicationMode model.replicationAttrs.modeForUrl)) as |replicationAction index|}}
```

Comment:
hmm, this conditional hurts my 🧠a little bit -- is there any way to make this a computed property, that way you can just do ``{{#each replicationAction as |replicationAction index|}}` or something similar?

Reply:
Updating this one now, or maybe even a reformat would help a bit!